tests/eval_expr.sysl

```diff
@@ -144,8 +144,10 @@ TransformApp:
       Nil = . -> <some> (:
         out = .
       )
-      stringInNull = "abc" in app.attrs.patterns
-      stringInList = "pqr" in ["abc", "def"]
+      stringInNull    = "abc" in app.attrs.patterns
+      stringInList    = "pqr" in ["abc", "def"]
+      stringNotInNull = "abc" !in app.attrs.patterns
+      stringNotInList = "pqr" !in ["abc", "def"]
```

Comment:
Would the following round out the testing:
```
stringInMap = "key1" in getMap("key")
stringNotInMap = "abc" !in getMap("key")
stringInSet = "a" in {"a","b","c"}
stringNotInSet = "d" !in {"a","b","c"}
```

Reply:
Ye, done.